cmd/livenessprobe/main.go

```diff
@@ -119,16 +125,37 @@ func acquireConnection(ctx context.Context, metricsManager metrics.CSIMetricsMan
 }
 
 func main() {
-	klog.InitFlags(nil)
+	var fs flag.FlagSet
+	klog.InitFlags(&fs)
+	fs.VisitAll(func(f *flag.Flag) {
+		switch f.Name {
+		case "logtostderr":
+			flag.CommandLine.Var(f.Value, f.Name, f.Usage)
+		}
+	})
```

Comment:
Can you please add a comment why do we need this VisitAll?

Reply:
Thank you for pointing it out. I've added a comment to the source code based on your feedback.